submissions/webNeat/solution.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+if ($argc < 2) {
+  $command = "php"
```

Comment:
@aboullaite 
I want to pass some optimization flags to the `php` command when running my script (to enable JIT compiler).

So when the runner executes my script with `php sol.php`

My script will run itself again with additional flags:
```
php -d opcache.enable_cli=1 -d opcache.jit_buffer_size=128M -d opcache.jit=1235 -d memory_limit=20G -f sol.php -- jit
```
And wait for this second command to finish before exiting.

Reply:
Understood but since we are not allowing any custom config/flags for other languages, I think it's unfair to allow it here.

Reply:
Does it make more sense for you to add it to [the runner](https://github.com/geeksblabla/blarun/blob/main/src/main.rs) ? because it already applies optimization flags for C++ and Rust.

Reply:
feel free to open a PR in the runner and we will try to discuss it in the team

Reply:
Ok, I will remove this optimization for now and make the PR to the runner.